docs/frontend/javascript-approaches/packs.md

```diff
@@ -3,6 +3,12 @@ title: Pack Files & Webpacker
 sidebar_position: 2
 ---
 
+:::important
+
+Any refactored or new functionality that lives in the asset pipeline, should now be moved to Packs and Webpacker.
```

Comment:
Base don my first comment maybe we can update this a bit, I'm not sure that I completely follow what this means.